### PR TITLE
Add: translation for Bengali (bn) locales language

### PR DESCRIPTION
**Pull request by gitworkflows**
- [*] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [*] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _data/locales/bn.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    related_guides: সম্পর্কিত গাইড
+  footer:
+    contribute:
+      heading: অবদান
+      description: পরামর্শ দিতে চান? এই কনটেন্টটি ওপেন সোর্স, আমাদের এটি উন্নত করতে সহায়তা করুন।
+      button: অবদান
+    subscribe:
+      heading: যোগাযোগে থাকুন
+      description: GitHub এর সর্বশেষ ওপেন সোর্স টিপস এবং রিসোর্স সবার আগে জানতে চান?
+      label: ইমেল ঠিকানা
```
Comment (adilshahriar) [inline _data/locales/bn.yml:21]: ইমেইল

**Comment by Qamardxb** [comment]
banglawiki:locales/bengali

**Comment by evifiller** [comment]
banglawiki:locales/bengali

**Review by khalilmosavi — COMMENTED** [review]
_data/locales/bn.yml

**Comment by Qamardxb** [comment]
banglawiki:locales/bengali

**Comment by BlueButterflies** [comment]
Check the details of GitHub Actions CI.Under the word TEST you will find all the errors to correct.

**Comment by SadeQuin50** [comment]
Thank u

**Review by Rifat-Hasan-BD — COMMENTED** [review]
As a native Bengali speaker, I can confirm that the Bengali translation in this file is machine-generated and of poor quality. It's difficult to understand. Please consider using a professional translator or improving the machine translation quality.

**Comment by jmeridth** [comment]
> As a native Bengali speaker, I can confirm that the Bengali translation in this file is machine-generated and of poor quality. It's difficult to understand. Please consider using a professional translator or improving the machine translation quality.

@Rifat-Hasan-BD thank you for this feedback.  As this is open source, hiring a professional translator is mostly unlikely.  Would you consider helping us out with the translation?

**Comment by ahpook** [comment]
Gonna close this per the comment from @Rifat-Hasan-BD

**Pull request closed without merging**